### PR TITLE
Clean-up blackwhitelist.py

### PR DESCRIPTION
**Pull request by labrys**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Always make sure the PR has a clear description of why,how, and what your trying to fix. The same goes for improvements or new features. Also if you refering to an issue, make sure a summary is available in the PR.
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by p0psicles** [comment]
Please provide some more information in the PR.
Also describe how you where able to reproduce and test the issue.

The PR is concluded when your able to at least check all checks.

**Comment by labrys** [comment]
@p0psicles sorry, details in commit must have gotten dumped during my squash.  This is just a style cleanup for things such as quotes, string formatting, indentation, and using unicode_literals.

File: sickbeard/blackandwhitelist.py [hunk 1]
```diff
@@ -82,7 +90,11 @@ def _del_all_keywords(self, table):
         :param table: SQL table remove keywords from
         """
         main_db_con = db.DBConnection()
-        main_db_con.action('DELETE FROM [' + table + '] WHERE show_id = ?', [self.show_id])
+        main_db_con.action(
+            b'DELETE FROM [{table}}] '
```
Comment (p0psicles) [inline sickbeard/blackandwhitelist.py]: Error in { }}

Comment (labrys) [inline sickbeard/blackandwhitelist.py]: doh!

**Comment by p0psicles** [comment]
I can't connect to anidb udp interface from here. Will check further tonight. Consider it as reviewed.

**Comment by p0psicles** [comment]
Approved

**Comment by p0psicles** [comment]
Any specific reason for this to be on hold?

**Comment by fernandog** [comment]
As I said in IRC: it depend on your PR as you made it against this branch. Need to wait you finish yours. Did you see I added the reference? Any problem about that?

**Comment by p0psicles** [comment]
Ooh sorry. Thats A misunderstanding. I'll PR that one against dev. Imo this one can be merged. The other PR was purely for discussion

**Comment by p0psicles** [comment]
https://github.com/pymedusa/SickRage/issues/392

**Pull request merged**